### PR TITLE
New livebook explaining pipes

### PR DESCRIPTION
**Pull request by resterle**

### CODE REVIEW

File: livebooks/manage_the_zoo_with_pipes.livemd [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Manage the zoo with pipes
+
+## Try it yourself!
+
+[![Run in Livebook](https://livebook.dev/badge/v1/black.svg)](https://livebook.dev/run?url=https%3A%2F%2Fgithub.com%2Finspired-consulting%2FElixir-Meetup-Cologne%2Fblob%2Fmain%2Flivebooks%2Fmanage_the_zoo_with_pipes.livemd)
```
Comment (curiosdevcookie) [inline livebooks/manage_the_zoo_with_pipes.livemd:5]: I can't run the livebook 😕

image

File: livebooks/manage_the_zoo_with_pipes.livemd [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+end
+```
+
+## We have the following animals in our zoo
+
+```elixir
+animals = ["🦉", "🦉", "🦩", "🦩", "🦩", "🐻", "🐻"]
+```
+
+## How many brown birds there are?
```
Comment (curiosdevcookie) [inline livebooks/manage_the_zoo_with_pipes.livemd]: Word order: How many brown birds … are there?

File: livebooks/manage_the_zoo_with_pipes.livemd [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+##### Count the brown birds and print the result
+
+```elixir
+"Number of brown birds: #{length(brown_birds)}"
+```
+
+## The pipe operator
+
+The pipe operator is writen as `|>`. It links the result of the expression on the left site of the operator to the function on the left site. It does this by passing the result of the expression to the functions as first parameter. The following:
```
Comment (curiosdevcookie) [inline livebooks/manage_the_zoo_with_pipes.livemd]: written

Comment (curiosdevcookie) [reply livebooks/manage_the_zoo_with_pipes.livemd]: the expression on the left … side

Comment (curiosdevcookie) [reply livebooks/manage_the_zoo_with_pipes.livemd]: to the function on the … right side?

File: livebooks/manage_the_zoo_with_pipes.livemd [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+Is therefore the same as this:
+
+```elixir
+"These animals live in the zoo #{animals}"
+|> IO.puts()
+```
+
+## Compose the steps using pipes
+
+With this in mind we can compose the steps to determine the number of brown birds with pipes.
```
Comment (curiosdevcookie) [inline livebooks/manage_the_zoo_with_pipes.livemd]: With this in mind,

File: livebooks/manage_the_zoo_with_pipes.livemd [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+animals
+|> Enum.filter(is_bird?)
+|> Enum.filter(is_brown?)
+|> length()
+|> then(fn number_of_brown_birds -> "Number of brown birds: #{number_of_brown_birds}" end)
+```
+
+## Compose with pipes
+
+Pipes are a great tool to compose functions that solve small proplems to solve bigger problems while keeping the code clean and easy to understand.
```
Comment (curiosdevcookie) [inline livebooks/manage_the_zoo_with_pipes.livemd]: solve small … problems

**Review by curiosdevcookie — DISMISSED** [review]
Great and cute blog post!

File: livebooks/manage_the_zoo_with_pipes.livemd (hunk 1 above)
Comment (resterle) [reply livebooks/manage_the_zoo_with_pipes.livemd:5]: Because it was not merged yet :)

File: livebooks/manage_the_zoo_with_pipes.livemd [hunk 6]
```diff
@@ -75,7 +75,7 @@ animals
 
 ## Compose with pipes
 
-Pipes are a great tool to compose functions that solve small proplems to solve bigger problems while keeping the code clean and easy to understand.
+Pipes are a great tool to compose functions that solve small prorblems to solve bigger problems while keeping the code clean and easy to understand.
```
Comment (curiosdevcookie) [inline livebooks/manage_the_zoo_with_pipes.livemd]: solve small … problems

**Review by curiosdevcookie — DISMISSED** [review]

**Pull request merged**